src/Model/Table/LazyTableTrait.php

```diff
@@ -1,4 +1,6 @@
 <?php
+declare(strict_types=1);
```

Comment:
do you want to apply this PR first on `4.x` branch ?
`master` can not receive this line

Reply:
I'm just a dummy and had the wrong composer dependencies installed and my autoformatter did the wrong thing.